xwiki-platform-core/xwiki-platform-web/src/main/webapp/templates/accessdenied.vm

```diff
@@ -12,15 +12,19 @@ $services.localization.render('notallowed')
 #template("startpage.vm")
 <div class="main layoutsubsection">
 <div id="mainContentArea">
-#if ($xwiki.isReadOnly())
-#xwikimessageboxstart($services.localization.render('error') $services.localization.render('readonly'))
+#if($xwiki.exists("XWiki.AccessDenied"))
```

Comment:
I think a space is missing after the `#if` and before the `(`. At least that's what you added below at line 18 ;)